constants.py

```diff
@@ -676,6 +679,16 @@
 kOffsetDistanceRange = 1
 """meters"""
 
+kTimeOfFlightList = [
```

Comment:
What is this?

Reply:
Amount of time in seconds it takes for the ball to travel to hub at various distances